plugins/importtext_procedure.cc

```diff
@@ -697,11 +698,12 @@ struct ImportTextProcedureWorkInstance
 	        }             
 	    }
 	    
-	    // Early check for duplicate column names
+	    // Early check for duplicate column names in input
+      ML::Lightweight_Hash<ColumnHash, int> inputColumnIndex;
```

Comment:
There are two hash structures and one vector structure to keep the column names.  I don't have a complete view of the code but that seem too many.  Can we remove one or two?